packages/presentation-exchange/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './PresentationExchangeError'
```

Comment:
there now seems to be logic in core, as well as in a separate presentation exchange package